### DISCUSSION TITLE
Can't instal on MacBook M1 Air

### DISCUSSION THREAD

**Discussion by joneser13** [koush/scrypted#1126] [Q&A]
I origannaly installed homebridge on my macbook air just so i could use my ring door bell with Homekit. Unfortuntly i found out it doesn't support HSV and the articles online said to install Scrypted. I'm pretty tech savy but for some reason I can not get it to install. I installed homebrew and then copied the link into the terminal. Any help would be much appreicated.

**Comment by jkpe** [comment]
The recommended installation method for Mac is the desktop app

https://docs.scrypted.app/installation.html#desktop-app
